tests/epyccel/test_base.py

```diff
@@ -69,6 +69,11 @@ def test_compare_is_not_nil():
 
 def test_cast_int():
     compare_epyccel(base.cast_int, 4)
+    compare_epyccel(base.cast_float_to_int, 4.5)
```

Comment:
As you are testing casting, it would be good to also add a test to show that the return type is correct (see https://github.com/pyccel/pyccel/blob/master/tests/epyccel/recognised_functions/test_math_funcs.py#L41)